src/Http/Http.Features/src/HeaderNames.cs

```diff
@@ -283,9 +283,15 @@ public static class HeaderNames
         /// <summary>Gets the <c>X-Frame-Options</c> HTTP header name.</summary>
         public static readonly string XFrameOptions = "X-Frame-Options";
 
+        /// <summary>Gets the <c>X-Powered-By</c> HTTP header name.</summary>
+        public static readonly string XPoweredBy = "X-Powered-By";
+
         /// <summary>Gets the <c>X-Requested-With</c> HTTP header name.</summary>
         public static readonly string XRequestedWith = "X-Requested-With";
 
+        /// <summary>Gets the <c>X-UA-Compatible</c> HTTP header name.</summary>
+        public static readonly string XUaCompatible = "X-UA-Compatible";
```

Comment:
Just noting here - missed before the other PRs went through:
With the exception of  `XXssProtection`, all other headers match the casing of upper case for acronyms. They're the same casing as their actual header name (e.g. `WWWAuthenticate` above) and seem more correct. I'm not sure if `XXssProtection` is released currently, but if not then changing it to `XXSSProtection` and upper-casing these additions (e.g. `XUACompatible`) would be more consistent.

I know this is a nit in the grand scheme, but it is an API break to ever fix it so...mentioning!

Reply:
Changed!